### DISCUSSION TITLE
[Bug Report] ffmpeg error

### DISCUSSION THREAD

**Discussion by trojanabanana** [stashapp/stash#5209] [Q&A]
I am getting the "missing ffmpeg and/or ffprobe" despite both manually and letting stashapp download ffmpeg, I unzipped it, and have manually changed the "application path(s)" inside of stash to match that of the correct folder "C:\Users\username\.stash\ffprobe.exe". I did verify that ffmpeg is working using cmd(opening cmd and typing ffmpeg) it tells me the version and such. But stash still says its missing ffmpeg. Another bit that might be useful to anyone willing to help me is that when I change the executable path(inside stash->systems) I get the error "invalid ffmpeg path: error running ffmpeg: open NUL: The system cannot find the file specified." 

Im unsure as to how to reproduce this because as far as Im aware I've done all the right steps at least according to the guy in the discord who tried to help me. 

stash version: v0.26.1 build hash d1998cb5

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome
 - Version 126.0.6478.63

Sorry if this is not the right venue to ask for troubleshooting, I'm now at the bottom of this wondering if there is another more apt location to ask for assistance on github.  Thanks for any potential assistance this post may garner.

**Comment by WithoutPants** [comment]
Sorry for missing this. Were you able to resolve it? 

It's a difficult one to troubleshoot remotely. Can you paste up the exact paths of your `ffmpeg.exe` and `ffprobe.exe` files, the values of `ffmpeg_path` and `ffprobe_path` (if present) in your `config.yml` file? And also please paste the exact output of the log file when setting these (or when scanning).
